test/run_outer_tests.py

```diff
@@ -158,6 +158,7 @@ def main():
         Command(functional_base_cmd + ["-z"], fail_cmd).run(retcode=127 if args_disabled else 1)
         Command(functional_base_cmd + ["-h"], fail_cmd).run(retcode=127 if args_disabled else 0)
         Command(functional_base_cmd + ["zzzz"], fail_cmd).run(retcode=127)
+        Command(functional_base_cmd + ["-w"], fail_cmd).run(retcode=127)
```

Comment:
This should be `retcode=127 if args_disabled else 0`: the `MINIMAL=1` build does not accept arguments at all (it's the one Docker uses for their embedded version of Tini).